source/Launchpad-Configuration/String.extension.st

```diff
@@ -0,0 +1,7 @@
+Extension { #name : #String }
+
+{ #category : #'*Launchpad-Configuration' }
+String >> asBoolean [
+
+	^ self = 'true'
```

Comment:
I think it's better to explicitly check for `true` and `false` and fail otherwise. I don't like the idea that the default behavior if I mistype the value is to consider it false.